pkg/monitoring/migration/prometheus.go

```diff
@@ -0,0 +1,98 @@
+package migration
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	k8sv1 "k8s.io/api/core/v1"
+
+	virtv1 "kubevirt.io/api/core/v1"
+)
+
+var (
+	migrationsLabels = []string{"vmi", "source", "target"}
+
+	currentPendingMigrations = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "kubevirt_migrations_pending",
+			Help: "Number of current pending migrations.",
+		},
+		migrationsLabels,
+	)
+
+	currentRunningMigrations = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "kubevirt_migrations_running",
+			Help: "Number of current running migrations.",
+		},
+		migrationsLabels,
+	)
+
+	migrationsSucceededTotal = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "kubevirt_migrations_succeeded_total",
+			Help: "Number of migrations successfully executed.",
+		},
+		migrationsLabels,
+	)
+
+	migrationsFailedTotal = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "kubevirt_migrations_failed_total",
+			Help: "Number of failed migrations.",
+		},
+		migrationsLabels,
+	)
+)
+
+func RegisterMigrationMetrics() {
```

Comment:
It'd be great if you can add a log entry here

Reply:
added